examples/.cargo/config.toml

```diff
@@ -4,11 +4,12 @@ esp32c2 = "run --release --features=esp32c2 --target=riscv32imc-unknown-none-elf
 esp32c3 = "run --release --features=esp32c3 --target=riscv32imc-unknown-none-elf"
 esp32c6 = "run --release --features=esp32c6 --target=riscv32imac-unknown-none-elf"
 esp32h2 = "run --release --features=esp32h2 --target=riscv32imac-unknown-none-elf"
+esp32p4 = "run --release --features=esp32p4 --target=riscv32imafc-unknown-none-elf"
 esp32s2 = "run --release --features=esp32s2 --target=xtensa-esp32s2-none-elf"
 esp32s3 = "run --release --features=esp32s3 --target=xtensa-esp32s3-none-elf"
 
 [target.'cfg(target_arch = "riscv32")']
-runner    = "espflash flash --monitor"
+runner    = "espflash flash --monitor --no-stub"
```

Comment:
Nit pick: We probably still want to use the stub for other chips, can we duplicate this and specify `target = "riscv32imafc-unknown-none-elf"` to only pass `--no-stub` on the P4?

Reply:
Honestly, I'm not even sure why is it there. 
It doesn't affect a program run/flash in any sense :shrug: